adamsjr8576/swapi-trivia
# Testing gi

#### What does this PR do?
- Adds testing for Nav, Login, MovieContainer, and MoiveCard
- Adds PropTypes to the above files
- Adds snapshots for above files

#### How to test?
- Once you have these changes on your local just run `npm test` to see the new tests

#### Issues
- I closed out the last card on the project board that was related to testing for the above components. 
-  I also closed out the PropTypes card since yours are finished as well.

#### Other Notes
- We are still only at about 30% test coverage, as you mentioned a lot of that will change once we test fetch. After we finish that we can reassess what else to test?

@adamsjr8576
